packages/@aws-cdk/aws-ecs-patterns/test/ec2/l3s-v2.test.ts

```diff
@@ -1094,9 +1091,8 @@ describe('When Network Load Balancer', () => {
     });
 
     // THEN - stack does not contain a LaunchConfiguration
-    const template = SynthUtils.synthesize(stack, { skipValidation: true });
-    expect(template).not.toHaveResource('AWS::AutoScaling::LaunchConfiguration');
-    expect(() => SynthUtils.synthesize(stack)).toThrow();
+    // Template.fromStack(stack).resourceCountIs('AWS::AutoScaling::LaunchConfiguration', 0);
```

Comment:
I am not sure what to do here. Previously when I migrated this from nodeunit to jest @nija-at said that we would need to make the validation pass at this point. However, the validation error I am getting is:

```bash
Cluster for this service needs Ec2 capacity. Call addXxxCapacity() on the cluster
```

The test is failing validation by nature however. Its name is "set vpc instead of cluster". The error stems from there not being a cluster on the `NetworkMultipleTargetGroupsEc2Service`. Thus, I am not sure how to pass the validation without altering the tests' original intention.

I guess our options are to either 1) support `skipValidation` in the `assertions` module or 2) modify this test by adding a cluster and losing fidelity.

This happens in 3 places. Twice in `l3s-v2.test.ts` and once in `l3.test.ts`.

Reply:
Sounds like a case where the code and/or test needs to be fixed.

I don't see the value in supporting skipValidation. 
We should be testing the code path users will use, and they are not (rather, should not) skip validations.
We should not add a feature that can be misused to workaround a bug in our construct library.